ui/src/services/WishDetailsService.js

```diff
@@ -6,10 +6,19 @@ const getWishDetails = async(id) => {
   return axios
     .get(`${expressDomain}/wishes/${id}`)
     .then(response => {
-      return response
+      return response.data
+    })
+}
+
+const makeAWish = async(wish) => {
+  return axios
```

Comment:
semi nit picky, but why mix the `.then()` syntax with async/await?

Reply:
Good point, was just due to the example I was taking from, I’ll fix in the morning

Reply:
Fixed this in latest commit